automl/google/cloud/automl_v1beta1/tables/tables_client.py

```diff
@@ -290,6 +292,29 @@ def __model_name_from_args(
             )
         return model_name
 
+    def __log_operation_info(self, message, op):
+        name = "UNKNOWN"
+        try:
+            if (
+                op is not None
+                and op.operation is not None
+                and op.operation.name is not None
+            ):
+                name = op.operation.name
+        except:
```

Comment:
The linter is throwing an error about the bare except. What kind of exception do you anticipate being thrown here? 

`google/cloud/automl_v1beta1/tables/tables_client.py:304:9: E722 do not use bare 'except'`

Reply:
`AttributeError`, in the case where one of the checks for `None` fails leaving the operation running but without handing it back to the user.